pkg/lib/cockpit-components-dialog.css

```diff
@@ -0,0 +1,4 @@
+#cockpit_modal_dialog .modal-body {
+    max-height: calc(75vh - 10rem);
+    overflow: auto;
+}
```

Comment:
Scrolling the entire dialog body seems a bit odd; don't we want to just scroll the package list?

Reply:
Hmm, @garrett?